src/java/org/apache/cassandra/utils/ByteBufferUtil.java

```diff
@@ -955,6 +955,18 @@ public static boolean contains(ByteBuffer buffer, ByteBuffer subBuffer)
         }
         return false;
     }
+    /**
+     * Check is the given buffer matches with the second buffer
+     *
+     * @param buffer1 The first buffer
+     * @param buffer2 The second buffer.
+     *
+     * @return true if both the buffers are equal, false otherwise.
+     */
+    public static boolean matches(ByteBuffer buffer1, ByteBuffer buffer2)
+    {
+        return buffer1.equals(buffer2);
+    }
```

Comment:
nit: I'd probably just inline this in `Expression` rather than having a one-line utility method.

Reply:
done. Thanks